### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
This pull request adds the build instructions for both windows and unix like operating systems.

Instructions for:
- [x] Unix Like Operating Systems
- [x] Windows

Also using:
- [x] `GNU make`
- [x] `cmake`

### CODE REVIEW

**Comment by Calamity210** [comment]
This is assuming everyone is using cmake. Both the gcc and mingw installs are redundant, please add steps for make as well

**Review by Calamity210 — CHANGES_REQUESTED** [review]
Take a look at the comment

**Comment by Andy-Python-Programmer** [comment]
@Calamity210 I think that `mingw` install can be important as the build will not work in windows build tools using cl.

**Comment by Calamity210** [comment]
Afaik mingw doesnt ship cmake

**Comment by Andy-Python-Programmer** [comment]
@Calamity210 I know but cmake will need mingw to work.

**Comment by Andy-Python-Programmer** [comment]
Added build instructions for using `make` @Calamity210

File: README.md [hunk 1]
```diff
@@ -4,5 +4,77 @@
...
+3. Build and run:
+```sh
+	$ make
+	$ make debug
+	$ cd build
+	$ ./senegal
+```
+
+## Building on Windows
+1. Make you have installed the dependencies:
```
Comment (MrBirb) [inline README.md:55]: nit: Make sure*

**Comment by Calamity210** [comment]
@Andy-Python-Programmer im going to hold off on reviewing this as there are going to be some changes in the build process soon

**Comment by Calamity210** [comment]
@Andy-Python-Programmer please take a look at 022006a

**Review by Calamity210 — CHANGES_REQUESTED** [review]

**Comment by Calamity210** [comment]
Closing since #14 makes it simpler to build and run senegal

**Pull request closed without merging**